components/security/authorization/pom.xml

```diff
@@ -47,6 +47,18 @@
       <artifactId>xwiki-platform-users-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>phenotips-constants</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
```

Comment:
This is a [runtime dependency of the `XWiki` class](https://github.com/xwiki/xwiki-platform/blob/xwiki-platform-7.0.1/xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWiki.java#L64), and since it is marked as a `provided` dependency of oldcore, it is not transitively included in the classpath.